test/e2e/selinux_mount_readiness.go

```
@@ -0,0 +1,385 @@
+		return 0, fmt.Errorf("failed to parse prometheus response: %w", err)
+	}
+	if queryResp.Status != "success" {
+		return 0, fmt.Errorf("prometheus query returned status: %s", queryResp.Status)
+	}
+
+	var total float64
+	for _, result := range queryResp.Data.Result {
+		if len(result.Value) >= 2 {
+			if valStr, ok := result.Value[1].(string); ok {
+				var val float64
+				fmt.Sscanf(valStr, "%f", &val)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Sscanf` is not checked

(errcheck)